### PR TITLE
My First Code

### PR DESCRIPTION
**Pull request by imaadkhan87**
# A
### C

:+1:

### CODE REVIEW

File: HelloWorld.java [hunk 1]
```diff
@@ -0,0 +1,6 @@
+//This is a class
+public class A {
+	public static void main(String[] args) {
+		System.out.println("Hello World");
+	}
+}
```
Comment (imaadkhan87) [inline HelloWorld.java]: No newline
@SherryShuranHuang

Comment (imaadkhan87) [inline HelloWorld.java]: Can you fix this for me?
@SherryShuranHuang

Comment (SherryShuranHuang) [inline HelloWorld.java]: No, unless a new redmine is added

**Comment by SherryShuranHuang** [comment]
Fixed

**Comment by imaadkhan87** [comment]
Nice job @SherryShuranHuang

**Pull request merged**